pkg/agent/utils.go

```diff
@@ -311,7 +315,7 @@ func getOrderedKubeletConfigFlagWithCustomConfigurationString(customConfig, defa
 
 	keys := []string{}
 	for key := range config {
-		if !datamodel.CommandLineOmittedKubeletConfigFlags[key] {
+		if !datamodel.GetCommandLineOmittedKubeletConfigFlags()[key] {
```

Comment:
probably a func call once before the loop makes sense

Reply:
It seems to be running per key that is in config. Is that not what should happen?